server/middleware/setJwt.js

```diff
@@ -6,7 +6,7 @@ const setJwtOnAccessToken = (request, response) => {
 	const token = jwt.sign({ username, password }, PRIVATE_KEY);
 
 	response.cookie('access_token', token, { httpOnly: true });
-	response.send('We have made a JWT for you!');
+	response.redirect('/userAccountPage');
```

Comment:
Nice, I like this.

In terms of UX, perhaps it would make sense to redirect to the home page?

Reply:
Happy to do that but I'll do it in a later PR. When I add redirect for failure, I'll add a redirect for success to the homepage as well.